content/kubermatic/master/tutorials/12-audit-logs/_index.en.md

```diff
@@ -0,0 +1,59 @@
++++
+title = "Audit Logging"
+date = 2020-04-02T14:50:15+02:00
+weight = 120
++++
+
+Audit Logging is one of key security features provided by Kubernetes. Once enabled in kube-api, it provides a chronological record of operations performed on the clusters by users, administrators and other cluster components.
```

Comment:
"cluster" in singular please. The feature has to be enabled for each cluster independently.